prdoc/pr_7026.prdoc

```diff
@@ -0,0 +1,17 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: [pallet-broker] add extrinsic to remove a lease
+
+doc:
+  - audience: Runtime User
+    description: |
+      A new `remove_lease` extrinsic is introduced to allow a lease to be removed immediately.
```

Comment:
```suggestion
      A new `remove_lease` extrinsic is introduced to the broker pallet to allow a lease to be removed by the root origin.
```
"Immediately" might be misleading in this description, as it's still on core due to its lease but just missing from the leases array.